generator/net_snmp.go

```diff
@@ -18,10 +18,23 @@ package main
 #cgo CFLAGS: -I/usr/local/include
 #include <net-snmp/net-snmp-config.h>
 #include <net-snmp/mib_api.h>
+#include <net-snmp/types.h>
+#include <net-snmp/library/container.h>
+#include <net-snmp/agent/snmp_agent.h>
+#include <net-snmp/agent/snmp_vars.h>
```

Comment:
These headers don't seem to be required (tested and works correctly without them on 5.7.3, 5.8, 5.9.1).

Reply:
I had compile errors from Go if I removed them.

Reply:
Weird. What OS?

Just tested on multiple distributions (Debian/Ubuntu/Fedora/Arch/Gentoo) and it works on all of them without including the four headers mentioned above.